accelerator-core/src/main/java/com/tokbox/android/otsdkwrapper/wrapper/OTWrapper.java

```diff
@@ -212,7 +210,7 @@ public String getOwnConnId() {
    * @return the number of active connections.
    */
   public int getConnectionsCount() {
-    return mConnectionsCount;
+    return mSubscribers.size();
```

Comment:
@Raulkg I don't think the connectionsCount needs to return only number of active subscribers.
It should return the total of active connections in the session.

Reply:
Mabye we need to add `getSubscrribersCount()` method